### PR TITLE
Run4-hgx268 Add a new method to locate for a scintillator cell

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add a new method to locate for a scintillator cell 

#### PR validation:

Tested using a test code in a test area

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalGeometry/test/HGCalGeomLocatorTester.cc [hunk 1]
```diff
@@ -75,5 +81,36 @@ void HGCalGeomLocaterTester::doTest(const HGCalGeometry* geom, DetId::Detector d
...
+void HGCalGeomLocaterTester::doTestScintillator(const HGCalGeometry* geom, DetId::Detector det) {
+  const std::vector<DetId>& ids = geom->getValidDetIds();
+  std::cout << "doTest:: " << ids.size() << " valid ids for " << geom->cellElement() << std::endl;
+  const double tol = 0.5;
+  const unsigned int step = 10;
+  int all(0), good(0), bad(0);
+  for (unsigned int k = 0; k < ids.size(); k += step) {
+    ++all;
+    HGCScintillatorDetId id(ids[k]);
+    if ((id.iradiusAbs() != 7) && (id.iradiusAbs() != 41)) {
```
Comment (kpedro88) [inline Geometry/HGCalGeometry/test/HGCalGeomLocatorTester.cc:93]: why are 7 and 41 special?

Comment (bsunanda) [reply Geometry/HGCalGeometry/test/HGCalGeomLocatorTester.cc:93]: Currently there are issues - which I need to investigate. For 41 I need to store the largest R - for 7 most likely the issue was the readjustment of some R-values. I found out those issues when I introduced this new method. I shall work on them later

**Comment by kpedro88** [comment]
+upgrade

**Comment by cvuosalo** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**